examples/example-storybook-v6.4/lostpixel.config.ts

```diff
@@ -5,6 +5,7 @@ import { CustomProjectConfig } from 'lost-pixel';
 export const config: CustomProjectConfig = {
   storybookShots: {
     storybookUrl: './storybook-static',
+    elementLocator: '#root > *:first-child',
```

Comment:
Here it is already in storybookShots though 🤔

Reply:
yeah sorry, I actually did it first in inside storybookShots but changed 😄